### ISSUE TITLE
simpleT5 for Grammatical Error Correction

### ISSUE THREAD

**Issue by pradeepdev-1995** [Shivanandroy/simpleT5#15]
How to use simpleT5 for Grammatical Error Correction

**Comment by Shivanandroy** [comment]
Hi @pradeepdev-1995 You can use simpleT5 for grammar error correction or any other sequence to sequence tasks. You can label your grammatically incorrect sentences as `source_text` and correct sentences as `target_text` and use any of the t5/mT5/byT5 models for fine tuning.

**Comment by pradeepdev-1995** [comment]
@Shivanandroy  Thank you.
Shall I use the specifc GEC model -  flexudy/t5-small-wav2vec2-grammar-fixer  for fine-tuning using simpleT5?

**Comment by Shivanandroy** [comment]
You can use any T5 model of your choice. My recommendation will be to use you a pretrained t5/mt5 model and then fine tune it on your corpus.

**Comment by pradeepdev-1995** [comment]
@Shivanandroy okay

So If I  use simpleT5 for GEC then what prefix should I put like summarize in summarization

```
# T5 model expects a task related prefix: since it is a summarization task, we will add a prefix "summarize: "
df['source_text'] = "summarize: " + df['source_text']
```

**Comment by Shivanandroy** [comment]
refer to #2
